examples/config.yml

```diff
@@ -33,6 +33,8 @@ lighthouse:
     # The DNS host defines the IP to bind the dns listener to. This also allows binding to the nebula node IP.
     #host: 0.0.0.0
     #port: 53
+    #wildcard: true # Enable wildcarding on hosts e.g allowing my-app.my-host.nebula to resolve to the IP for my-host.nebula
+    #wildcard_limit: 5 # Limit of number of possible subdomains
```

Comment:
I'd rephrase this to something like `# The maximum supported subdomain depth` or something. "number of possible" subdomains sounds to me like it would limit the number of any level subdomains